web/customize_viewer.css

```diff
@@ -2,31 +2,160 @@
  * Custom css for our fork of pdf.js.
  */
 
+html body.light {
```

Comment:
We _could_ do something where we change the `.light` class to an id like `#iFixitTheme`. That may push its precedence up to turn the selector string from `html body.light {}` to `#iFixitTheme {}` which is simpler and makes a little more sense in my opinion.